SConstruct

```diff
@@ -1331,7 +1334,7 @@ if env["PLATFORM"] == "posix" :
 	pythonEnv.Append( CXXFLAGS = [ "-Wno-strict-aliasing" ] )
 
 # get the python link flags
-if pythonEnv["PYTHON_LINK_FLAGS"]=="" :
+if pythonEnv["PLATFORM"] != "win32" and pythonEnv["PYTHON_LINK_FLAGS"]=="" :
```

Comment:
Windows doesn't have python-config at all, but it also doesn't need any special flags to link correctly so it should be an empty string for Windows. Windows by default installs Python to C:\python27 (for example) but that can be overridden easily and in a dramatic twist it's up to the user to tell the installer to set the system-wide PATH variable to include their new Python installation.

So I think it's best to require the Python binary to be specified when building on Windows - I'll add a check for that and fail the build if it can't be found. I'll also clean this up some to make it more obvious what's happening on one platform vs the other.

Reply:
I cleaned this up a lot and added a new config test to make sure Python is working. Feel free to decline that addition if you think it's too much or it's causing problems.